Test/Test.cs

```diff
@@ -2591,7 +2592,34 @@ public void TestGetScoreRequest()
 
             Assert.Equal("https://api.sift.com/v205/users/123/score?api_key=345&abuse_types=payment_abuse,promotion_abuse&fields=SCORE_PERCENTILES",
                         Uri.UnescapeDataString(scoreRequest.Request.RequestUri!.ToString()));
-        }   
+        }
+
+        //TestWarnings
+        [Fact]
+        public void TestWarnings()

```

Comment:
Please add check that response contains warnings, so we sure deserialisation went fine. Basically response payload assertion like in other tests

Reply:
I have updated the method to include a check for warnings in the response payload to ensure deserialization is successful. Please have a look.